ligo/stablecoin/spec.ligo

```diff
@@ -0,0 +1,227 @@
+// SPDX-FileCopyrightText: 2020 tqtezos
+// SPDX-License-Identifier: MIT
+
+/*
+ * Stablecoin smart-contract
+ */
+
+/* ------------------------------------------------------------- */
+
+/*
+ * Parameter
+ */
+
+type tokenId is nat
+
+
+
+type transferDescriptor is record
```

Comment:
Maybe we should just call this `tansfer` or something like that because it seems that we have another `transferDescriptor` from FA2 as part of transfer hooks.

https://gitlab.com/tzip/tzip/-/blob/76d5f3791bfbfe3c9bf95ad5ec5fc6cbeeca2d0e/proposals/tzip-12/tzip-12.md#transfer-hook

Reply:
Good observation, I once already shot in my leg when stumbled with this naming